### ISSUE TITLE
Repo Health Checks

### ISSUE THREAD

**Issue by aphelionz** [orbitdb/welcome#68]
It would be great to have a matrix of all of our repos and their status in terms of code, readme, docs, tests, CI ,etc

**Comment by RichardLitt** [comment]
I've got this working using @mntnr/dashboard. What other specific checks would you like? Or should we overhaul this?

![Screen Shot 2019-09-11 at 15 58 17](https://user-images.githubusercontent.com/910753/64730409-46934c80-d4ad-11e9-934f-fa8800f7e570.png)

**Comment by aphelionz** [comment]
Awesome! I'd love to see checks for the existence of:

- a non-default "npm test" entry in package.json
- a "npm test:browser" entry in package.json
- a `benchmarks` folder
- a `examples` folder

Also: 
- current CI status (and a big red X if no `.circleci/config.yml` file exists)

Thanks for doing this!

**Comment by CSDUMMI** [comment]
Can this be closed?

**Comment by RichardLitt** [comment]
I'm unlikely to start this work up again. But I think it should stay open.

**Comment by aphelionz** [comment]
Is there code for this somewhere? The orbitdb org can fork / inherit it.

**Comment by RichardLitt** [comment]
Yeah! [Here](https://github.com/mntnr/dashboard). Could be forked. I don't think I had any specialty code written up for OrbitDB.

**Comment by aphelionz** [comment]
Awesome, I forked it over to the org. Thank you!
